test/migrations/20200401161126_move_nmbas_to_project_level.js

```diff
@@ -0,0 +1,448 @@
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        project_id: ids.draft,
+        status: 'submitted',
+        data: {
+          title: 'Draft Project',
+          protocols: [
+            {
+              title: 'protocol 1',
+              steps: [
+                {
+                  nmbas: true
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        project_id: ids.draft,
+        status: 'draft',
+        data: {
+          title: 'Draft Project',
+          protocols: [
+            {
+              title: 'protocol 1',
+              steps: [
+                {
+                  nmbas: true
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        project_id: ids.noNmbas,
+        status: 'granted',
+        data: {
+          title: 'Project no NMBAs',
+          protocols: [
+            {
+              title: 'protocol 1',
+              steps: [
+                {
+                  nmbas: false
+                }
+              ]
+            }
+          ]
+        }
+      }
+    ];
+
+    before(() => {
+      this.knex = db.init();
+    });
+
+    beforeEach(() => {
+      return Promise.resolve()
+        .then(() => db.clean(this.knex))
+        .then(() => this.knex('establishments').insert(establishment))
+        .then(() => this.knex('profiles').insert(licenceHolder))
+        .then(() => this.knex('projects').insert(projects))
+        .then(() => this.knex('project_versions').insert(versions));
+    });
+
+    afterEach(() => {
+      return db.clean(this.knex);
+    });
+
+    after(() => {
+      return this.knex.destroy();
+    });
+
+    it('is ok', () => {
+      assert.ok(true);
+    });
+
+    it('does not change any properties, only adds nmbas-used', () => {
+      return Promise.resolve()
+        .then(() => this.knex('project_versions').select('project_versions.*').orderBy('id'))
+        .then(before => {
+          return Promise.resolve()
+            .then(() => {
+              return up(this.knex);
+            })
+            .then(() => this.knex('project_versions').select('project_versions.*').orderBy('id'))
+            .then(after => {
+              const changes = diff(before, after);
+              assert.ok(changes.every(change => {
+                return change.kind === 'N' && change.path.pop() === 'nmbas-used';
+              }));
+            })
+        });
+    });
+
+    it('does not add nmbas-used to schema v0 versions', () => {
+      return up(this.knex)
+        .then(() => {
+          return this.knex('project_versions')
+            .join('projects', 'project_versions.project_id', 'projects.id')
+            .where('projects.schema_version', 0);
+        })
+        .then(versions => {
+          versions.forEach(version => {
+            assert.ok(!version.data['nmbas-used'], 'nmbas-used should not be added to legacy versions');
```

Comment:
You may want to check it doesn't have the property at all (i.e. equal `undefined`) this would pass if it was set to `false`